.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 *~
+.idea/
```

Comment:
I suggest adding this directory to a [global Git ignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file) instead. Arguably, the `*~` line should also not be here since it's not related to this repository.